.github/workflows/main.yml

```diff
@@ -29,4 +29,4 @@ jobs:
         run: npx prettier . --check
 
       - name: test
-        run: npx tree-sitter test
+        run: npx tree-sitter test --debug
```

Comment:
not sure --debug is needed here any longer

Reply:
of course not, I was just about to remove that :)